anti-evil-maid/sbin/anti-evil-maid-install

```diff
@@ -30,6 +30,10 @@ Arguments:
 
   -F: passed on to mkfs.ext4 (don't ask for confirmation, etc.)
 
+  -m: set up a multi-factor auth AEM media (EXPERIMENTAL)
+      Using time-based one time password and a LUKS key file, provides resistance
```

Comment:
Here it is: a051d2c. Sorry for misunderstanding (thought you meant line lengths in code, not script run logs).